ui/ts/pages/log.ts

```diff
@@ -119,11 +134,15 @@ module AdminViews {
       }
 
       export function view(ctrl: Controller): _mithril.MithrilVirtualElement {
-        let rows: _mithril.MithrilVirtualElement[] = [];
-        if (entries.result() != null) {
-          for (let i: number = 0; i < entries.result().length; i++) {
-            rows.push(_EntryRow(entries.result()[i], i));
-          }
+        let comparisonData: Table.TableData<LogEntry> = {
+          columns: ctrl.columns,
+          rows: entries.allEntries
+        };
+        let count: number;
+        if (entries.allEntries()) {
```

Comment:
This does indeed check for null.
Until after the first query returns, the results are null.